ap.md

```diff
@@ -0,0 +1,236 @@
+
+{% highlight javascript %}
+// respond to the request with our index.html page
+function showIndex (url, request, response) {
+  response.writeHead(200, {"Content-Type": "text/html"});
+  // use fs to read in index.html
+  fs.readFile(__dirname + '/index.html', function (err, content) {
+    if (err) {
+      throw err;
+    }
+
+    // serves the content of index.html read in by fs
+    response.end(content);
+  });
+}
+
+// toggle the led specified in the url and respond with its state
+function toggleLED (url, request, response) {
+  var indexRegex = /(\d)$/; // Regex to find the number at the end of the url
+  var result = indexRegex.exec(url); // capture the number, returns an array
+  var index = +result[1]; // grab the captured result from the array and make sure it's a Number
+
+  var led = tessel.led[index]; // use the index to refence the correct LED
+
+  // toggle the state of the led and call the callback after that's done
+  led.toggle(function (err) {
+    if (err) {
+      // log the error and send back a 500 (internal server error) response to the client
+      console.log(err);
+      response.writeHead(500, {"Content-Type": "application/json"});
+      response.end(JSON.stringify({error: err}));
+    } else {
+      // the led was successfully toggled, so respond with the state of the toggled led using led.isOn
+      response.writeHead(200, {"Content-Type": "application/json"});
+      response.end(JSON.stringify({on: led.isOn}));
+    }
+  });
+}
+{% endhighlight %}
+
+In our `showIndex` function, we grab the content of `index.html` using the Node.js `fs` module and respond to the web browser request with that content. In our toggleLED function, we use a Regex to search the url for the index of the led we want to toggle and pass that index into the `tessel.led` array. Now that we have the led we want, we toggle it and use the callback to respond to the server request with the current state of the led using `isOn` property.
+
+One last step is to create a `.tesselignore` file in the project directory and add any non-JavaScript files we want to deploy with `ap.js`. For this project, that means adding one line: `index.html`. 
+
+Finally, let's fire up our server again by running:
+
+`t2 run ap.js`
+
+Just like before, once you see the "Server running at http://192.168.1.101:8080/" message in your terminal, you should be able to connect to that url in the web browser of the device connected to the TesselRouter network. After connecting, you should see the index.html view we built earlier and, after clicking/tapping the buttons, see either the blue or green LEDs on the Tessel should be toggled on or off. 
+
+<video src="https://dl.dropboxusercontent.com/u/74986127/tessel-router-demo.mp4" controls loop class="small-12 medium-8" >Video not available at the moment.</video>
```

Comment:
Tips for making a better video (I know this is a pain, sorry..):

* Find a way to set up the camera and the Tessel so they are both completely still. If you don't have a tripod, you can make this work by balancing stacks of books and bringing over a desk light. Or call in help, maybe @sidwarkd would be up for testing this tutorial and filming it in action?
* CTRL + + a bunch of times so that the text is legible in the terminal and in the web browser